### PR TITLE
PNI CC Page - remaining work for Quiz Part 2

### PR DESCRIPTION
**Pull request by mmmavis**
Covers https://github.com/MozillaFoundation/foundation.mozilla.org/issues/11194

Review app: https://foundation-s-11194-pni--rsx0ei.herokuapp.com/en/privacynotincluded/articles/annual-consumer-creep-o-meter/

## Two things I hope we can consider to be post-MVP:

- Thumb-up emoji on the thank you screen is currently just a unicode. Could we replace it with an image instead?
- Newsletter form layout doesn't look like what's on the mockup. This is because I'm refraining from creating another variation of the newsletter form for now. Also I'm using our default disabled button style (light purple) instead of grey.

### CODE REVIEW

File: source/js/buyers-guide/components/product-quiz/product-quiz.jsx [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
+  }
+
+  getInitialState() {
+    return {
       selectedChoices: [],
       numBad: 0,
       score: 0,
-      showResults: false,
+      currentStep: 1,
+      showThankYou: false,
```
Comment (mmmavis) [inline source/js/buyers-guide/components/product-quiz/product-quiz.jsx:22]: @danielfmiranda if you wanna make the thank you screen show just for QAing purpose, you can set `currentStep` to 2 and `showThankYou` to false

File: source/js/buyers-guide/components/product-quiz/product-quiz.jsx [hunk 2]
```diff
@@ -28,6 +35,10 @@ class ProductQuiz extends Component {
   }
 
   componentDidUpdate(prevProps, prevState) {
+    if (this.state.showThankYou && !prevState.showThankYou) {
```
Comment (mmmavis) [inline source/js/buyers-guide/components/product-quiz/product-quiz.jsx:38]: oops. gotta remove this 😅

**Review by danielfmiranda — APPROVED** [review]
Amazing work! LGTM 🙌

**Comment by mmmavis** [comment]
@danielfmiranda thank you for jumping in so quickly. I'll merge this PR soon once CI finishes.

**Pull request merged**